eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/RestRelation.php

```diff
@@ -89,17 +89,25 @@ public function visit(Visitor $visitor, Generator $generator, $data)
      */
     protected function getRelationTypeString($relationType)
     {
-        switch ($relationType) {
-            case RelationValue::COMMON:
-                return 'COMMON';
-            case RelationValue::EMBED:
-                return 'EMBED';
-            case RelationValue::LINK:
-                return 'LINK';
-            case RelationValue::FIELD:
-                return 'ATTRIBUTE';
+        if ($relationType > 15 || $relationType < 1) {
```

Comment:
That's not the best choice maintenance-wise :-)
Please move it before `return` statement and change to `if (empty($relationTypeList))`

Reply:
I changed it